### ISSUE TITLE
Newlines are not preserved if a file contains multiple blocks

### ISSUE THREAD

**Issue by knrafto** [albert12132/templar#29]
If we have multiple blocks in a file, as in
```
-- <block q1>
CREATE TABLE q1 WHERE SELECT "q1";
-- </block q1>

-- <block q2>
CREATE TABLE q2 WHERE SELECT "q2";
-- </block q2>
```
then the output will be
```
CREATE TABLE q1 WHERE SELECT "q1";
CREATE TABLE q2 WHERE SELECT "q2";
```
but I expect that the newline between the blocks would be preserved, instead of putting the contents of the blocks one after the other.

Oddly enough, if there are two newlines, then two newlines appear in the output. For example,
```
-- <block q1>
CREATE TABLE q1 WHERE SELECT "q1";
-- </block q1>


-- <block q2>
CREATE TABLE q2 WHERE SELECT "q2";
-- </block q2>
```
is rendered into
```
CREATE TABLE q1 WHERE SELECT "q1";


CREATE TABLE q2 WHERE SELECT "q2";
```

Right now the only way to get the correct spacing is by adding the newline _inside_ the block, e.g.
```
-- <block q1>
CREATE TABLE q1 WHERE SELECT "q1";

-- </block q1>

-- <block q2>
CREATE TABLE q2 WHERE SELECT "q2";

-- </block q2>
```
but this is ugly and annoying.

This bug was introduced recently. It worked fine last semester, but we noticed it this semester when we tried generating solutions for hw01. The issue in the 61a repo is [here](https://github.com/Cal-CS-61A-Staff/berkeley-cs61a/issues/563).

**Comment by albert12132** [comment]
Should be fixed by 0af04c6. This is available in version 2.1.1.

**Comment by knrafto** [comment]
Thanks! Unfortunately it'll take a while to switch, since not we depend on the new behavior.
